serverless/knative_eventing/serverless-using-brokers.adoc

```diff
@@ -1,16 +1,24 @@
 [id="serverless-using-brokers"]
-= Using brokers with Knative Eventing
+= Using brokers as an event delivery mechanism
 include::modules/common-attributes.adoc[]
 include::modules/serverless-document-attributes.adoc[]
 :context: serverless-using-brokers
 
 toc::[]
 
-Knative Eventing uses the `default` broker unless otherwise specified.
+You can use an xref:../../serverless/event_sources/knative-event-sources.adoc#knative-event-sources[event source] or a xref:../../serverless/knative_eventing/serverless-sinkbinding.adoc#serverless-sinkbinding[SinkBinding], the `default` broker, and a trigger together to decouple applications that produce events from the destination configuration.
 
-If you have cluster administrator permissions, you can create the `default` broker automatically using namespace annotation.
+Events are sent from an event source to the broker as an HTTP POST.
+After events have entered the broker, they can be filtered and sent to an event sink using triggers.
+You can use a trigger to consume events from a broker based on event attributes. Your application will receive events as an HTTP POST.
```

Comment:
Please double-check the use of "consume" here.

> ! consume v
Do not use when a simpler term such as “use” would be more appropriate. Use only to mean “use up.”

And you need a noun after "POST."